pages/dkp/konvoy/2.2/storage/static-local-volume/index.md

```diff
@@ -0,0 +1,128 @@
+---
```

Comment:
Perhaps goes without saying, but please apply any agreed comments/changes from the 2.1 page above to this page too. :-)